tests/unit/Deserializers/EntityDeserializerTest.php

```diff
@@ -212,7 +212,43 @@ public function testInvalidSerialization( $serialization ) {
 
 	public function invalidDeserializationProvider() {
 		return array(
-			array(
+			'label with integer language code' => array(
+				array(
+					'type' => 'item',
+					'labels' => array(
+						8 => array(
+							'language' => 8,
+							'value' => 'Cat',
+						),
+					),
+				),
+			),
+			'label with integer value' => array(
+				array(
+					'type' => 'item',
+					'labels' => array(
+						'en' => array(
+							'language' => 'en',
+							'value' => 8,
+						),
+					),
+				),
+			),
+			'alias with interger language code' => array(
+				array(
+					'type' => 'item',
+					'aliases' => array(
+						8 =>
+							array(
+								array(
+									'language' =>  8,
```

Comment:
As above, please add a test with no array key (defaults to int 0) and a valid language code.